.github/workflows/build.yml

```
@@ -0,0 +1,42 @@
+name: CI
+on:
+  push:
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - run: sudo apt update && sudo apt install unzip zip
```

```suggestion
      - run: apt-get update && apt-get install -y unzip zip
```
apt is apparently not recommended for scripting and apt-get should be preferred, also the -y param is probably necessary